### PR TITLE
feat: CLI Landing page content 

### PR DESCRIPTION
**Pull request by DaydreamerUnicorn**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by BlueAcidFrog** [comment]
@DaydreamerUnicorn not sure what the changes are on this. I suggest we go over this together.

File: src/page-content/content-cli.tsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+      <span className={'is-mute-text'}> Infrastructure as code tool </span>
+    </>
+  ),
+  description:
+    'Automate your tasks, manage your projects, and interact with Ory Cloud seamlessly using Ory CLI.',
+  buttons: [
+    {
+      ctaTitle: 'Install Ory CLI',
+      style: 'filled',
+      to: '/guides/cli/installation'
```
Comment (BlueAcidFrog) [inline src/page-content/content-cli.tsx:38]:
```suggestion
      to: '/guides/cli/installation'
```

Comment (BlueAcidFrog) [reply src/page-content/content-cli.tsx:38]: This link does not work

File: src/page-content/content-cli.tsx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+)
+const ArrowRight = <i className={'ph-arrow-right-bold size16'} />
+
+export const seo: SeoProps = {
+  description: 'Use our CLI as the infrastructure for code tool for Ory Cloud automation. Install now to manage your workflows easily.',
+  title: 'Ory CLI - Infrastructure As Code Tool'
+}
+
+export const hero: HeroImageProps = { 
+  id: 'kratos.hero',
```
Comment (BlueAcidFrog) [inline src/page-content/content-cli.tsx:25]:
```suggestion
  id: 'kratos.hero',
```

Comment (BlueAcidFrog) [reply src/page-content/content-cli.tsx:25]: The image does not fit in with CLI

File: src/page-content/content-cli.tsx [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+          <>
+            Ory CLI works like no other. Just install and let it work the magic.
+          </>
+        )
+      },
+      {
+        title: <>Enjoy the ease</>,
+        description: (
+          <>
+           Install Ory and Run it. Now sit back and relax while the CLI got it.
```
Comment (BlueAcidFrog) [inline src/page-content/content-cli.tsx:120]:
```suggestion
           Install Ory and run it. Now sit back and relax while the CLI got it. 
```

File: src/page-content/content-cli.tsx [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+      <>
+        Automate and manage your projects through our CLI. Use it to automate GitHub actions, manage control and access to Git files, track changes, and much more.
+      </>
+    ),
+    features: [
+      {
+        title: <>Unique automation</>,
+        description: (
+          <>
+            Ory CLI works like no other. Just install and let it work the magic.
```
Comment (BlueAcidFrog) [inline src/page-content/content-cli.tsx:112]:
```suggestion
            Ory CLI works like no other. Just install and let it work the magic.
```

Comment (BlueAcidFrog) [reply src/page-content/content-cli.tsx:112]: This needs more substance. If there is no value to communicate we should not write something for the purpose of writing - be specific and precise

File: src/page-content/content-cli.tsx (hunk 3 above)
Comment (BlueAcidFrog) [inline src/page-content/content-cli.tsx:120]: same as above

File: src/page-content/content-cli.tsx [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+      style: 'filled',
+      to: '/guides/cli/installation'
+    }
+  ],
+  image: (
+    <img
+      className="responsive"
+      loading="lazy"
+      alt="Examples of task configuration and automation using Ory CLI"
+      src={Kratos}
```
Comment (BlueAcidFrog) [inline src/page-content/content-cli.tsx:46]: Does that not require to be changed as well?

**Review by BlueAcidFrog — CHANGES_REQUESTED** [review]
@DaydreamerUnicorn good in the meta description and SEO. Please review my comments - Thx

**Comment by BlueAcidFrog** [comment]
can you please add this part as well? 

Fast and Simple Use of CLI in Just 4 Steps

1. Install the Ory CLI
2. Save a file in Git (for example mfa: disabled)
3. See the changes made (who, what, and when) (For example John Doe enabled the MFA at 12:00 PM)
4. Write that in a file and let the CLI process it

And that’s it!

**Comment by DaydreamerUnicorn** [comment]
@vinckr, can you please edit the 'Kratos' bits in the VS code from the CLI landing page tonight?

File: src/page-content/content-cli.tsx (hunk 1 above)
Comment (RinorG) [reply src/page-content/content-cli.tsx:39]: where should it lead to?

File: src/page-content/content-cli.tsx (hunk 2 above)
Comment (RinorG) [reply src/page-content/content-cli.tsx:26]: Out of all alternatives currently in project, i think it's the best.

**Comment by DaydreamerUnicorn** [comment]
@jfcurran, please see that the icons need to be changed for the 'Fast and simple use of CLI' part.

File: src/page-content/content-cli.tsx [hunk 6]
```diff
@@ -173,31 +173,47 @@ export const usageList: FeaturelistProps = {
...
+            <> Once up and running, you can enable or disable Passwordless or Multi Factor Authentication on your project.
+            </>
+          ),
+          icon: Git
+        },
         {
           title: 'Process a file with commands',
           description: (
-            <> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            <> Use CLIs simplified commands to process a file or a project until you know you're done.
```
Comment (DaydreamerUnicorn) [inline src/page-content/content-cli.tsx:216]: Please edit **CLIs** to **CLI's**

**Comment by jfcurran** [comment]
please resolve the few changes that @BlueAcidFrog requested.

**Review by jfcurran — APPROVED** [review]
Some wording is repetitive, but this works

**Pull request merged**